### PR TITLE
add findNextRank to find higher rank than me

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #140** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/140" target="_blank">Graphite</a> 
    * **PR #141** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141" target="_blank">Graphite</a> 
      * **PR #142** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/142" target="_blank">Graphite</a> 
        * **PR #143** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/143" target="_blank">Graphite</a> 
          * **PR #145** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/145" target="_blank">Graphite</a> 
            * **PR #146** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/146" target="_blank">Graphite</a> 
              * **PR #147** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/147" target="_blank">Graphite</a> 
                * **PR #150** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/150" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/150?utm_source=stack-comment).

File: subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/out/persistence/AccessibilityRankRepository.kt [hunk 1]
```diff
@@ -27,6 +27,10 @@ class AccessibilityRankRepository(
         return queries.findByRank(rank).executeAsOneOrNull()?.toDomainModel()
     }
 
+    override fun findNextRank(rank: Long): AccessibilityRank? {
```
Comment (Zeniuus) [inline subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/out/persistence/AccessibilityRankRepository.kt:30]: next가 좀 애매한 것 같은데, upper나 higher 같은 용어는 어때요? 더 애매하게 느껴지나...

Comment (Zeniuus) [reply subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/out/persistence/AccessibilityRankRepository.kt:30]: 그리고 next 어감에 대한 것과 별개로 nearest 같은 단어가 들어가면 더 좋을 것 같습니다.

**Review by Zeniuus — APPROVED** [review]

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/150).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/150) rebased this pull request as part of a merge.

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/150).

File: subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/out/persistence/AccessibilityRankRepository.kt (hunk 1 above)
Comment (bellatoris) [reply subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/out/persistence/AccessibilityRankRepository.kt:30]: next가 바로 옆이라는 뜻이니 확실하지 않나요? ㅋㅋ

**Pull request merged**